update release team comms handbook (#1797)
kubernetes/sig-release: release-team/role-handbooks/communications/README.md

### Skills and Experience Required

- Strong written and verbal communications skills
- A working knowledge of Kubernetes concepts

> - Fundamental GitHub skills, experience with open source projects

Reviewer: Fundamental GitHub skills and experience with open source projects

- Experience with the Kubernetes release process enough to understand how communications milestones fit into the overall release
- Project management experience is helpful
- Existing relationships with the CNCF, relevant media personnel and outlets, Kubernetes project managers, and vendor stakeholders is helpful

### Expected Time Investment

> The Kubernetes release cycle spans 16 weeks, though may run longer. The typical workload for the communications team is very light during the first few weeks. In the later weeks of the release cycle, the workload can become heavy, and often work can continue on a few weeks after release.

Reviewer 2: ```suggestion
The Kubernetes release cycle spans 15 weeks, though may run longer. The typical workload for the communications team is very light during the first few weeks. In the later weeks of the release cycle, the workload can become heavy, and often work can continue on a few weeks after release.
```
We state 15 weeks in the [release cadence kep](https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#summary), the end of the year release tends to be 16 weeks

### Release blog

The Communications Coordinator along with the Comms shadows are responsible for authoring the official Kubernetes Release Blog. This blog is the official statement of release. The release blog is the primary vehicle by which the release team communicates the major themes, known issues, and other aspects of the release to the community. 

Start the draft with the team around week 12, striking a balance between the release content being fairly well set and having enough to time author the blog and have it reviewed. Ahead of review, file a PR on k/website and cc the release lead and other stakeholders as reviewers.

It's up to you and your team regarding how best to do this, typically it works well to assign sections to different team members (e.g. Major Themes, User Highlights, Ecosystem Updates, etc) and have the lead pull it all together and manage the PR and reviews.

### Feature blogs

Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and authored by enhancement developers and others close to the features.

> The first feature blog typically goes out on release day alongside or shortly after the release blog, and then are published one-at-a-time, typically 2-3 a week. The Comms team establishes the publication schedule. Note that blog PRs in k/website are dated, and automation will publish future-dated entries. This enables a PR process decoupled from blog publication date.

Author: I changed this to "two to three posts weekly"

As feature blogs are opted in, assign them to shadows and yourself for tracking and facilitation. The responsibility is to ensure the blog authors have the resources and information they need, including editorial and tech reviews once ready.

Work closely with the SIG Docs Blogs team (connecting on `#sig-docs-blogs` and by attending meetings), as they are typically available for editorial reviews. Share with them the feature blog schedule and updates throughout the cycle.

For tech reviews, reach out to authors and the sponsoring SIG to organize at least one tech review per blog post.

### Mid-cycle deprecations and removals blog

> This blog is optional and will vary from release to release. Work with the rest of the release team ahead of the Code Freeze date to determine if a mid-cycle blog focused on feature deprecations and removals is warranted. If so, facilitate its creation and publication.

Reviewer 3: We could link to some previous examples, eg:
- https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/
- https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/

Reviewer: It looks like @jlbutler linked to the 1.22 deprecations doc further down the page.

Author: I moved the example up into this section, and used both of these.

If the release will deprecate important and commonly-used features (or simply a large number of features will be deprecated), consider publishing this blog. Also consider this when commonly-used features that have been deprecated are removed in a release.

> Regardless of reason, the timing for publication is ahead of the release, in order to inform the community and allow for preparation time. Start the discussion mid-cycle and well ahead of Code Freeze, and target publication for Code Freeze week.

Author: I think 1.24 is an outlier in this respect. By the time we get to enhancements freeze, we might have more to communicate as well.

### Press and release webinar

This is a light but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF, namely press release and interview scheduling around the release day, and the release webinar after the release.

To schedule the press briefings, you will be a liaison between the Release lead and the CNCF contacts. Send an email about a month ahead of the release and coordinate between the parties to get release day press events scheduled.

> To schedule the release webinar with the CNCF, use the Calendly link (below) to schedule a "live webinar". This will typically occur 3-4 weeks after the release, and is primarily presented by the Release lead and Enhancements lead. Often the Comms lead will also join the webinar. The format is open, but primarily the team walks through the enhancements and gives a sneak peek of what's coming in the next release.

Author: At this point, the "next release" is the next-next release :)

### Slack Channel

There is a channel on the Kubernetes Slack workspace, `#release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations. The Communications Coordinator (often referred to as the "Comms Lead") should post a status here at a regular cadence to keep release team members and other stakeholders informed.

## Release deliverables

Throughout the release cycle, the main Communications deliverables include:
* **Authoring the Kubernetes release blog.** The Communications team writes the release blog, which is the official announcement of the Kubernetes release.
* **Coordination and support of the feature blog series.** SIGs opt in to author feature blogs for the release. These blogs are written by technical owners, and the Communications team supports authors from the early stages through facilitating editorial and tech reviews as well as publication.

> * **Coordination and support of an optional Deprecations and Removals blog.** Depending upon the content of a given release, it may be necessary to prepare the community for upcoming deprecations and removals. This is decided per release cycle around the time of Code Freeze.

Reviewer 4: Just a few thoughts around the deprecations and comms - I'm not sure if its worth calling out explicitly, but around those I would engage with the contributor comms team and the CNCF (the contributor comms team might be able to facilitate that). For larger deprecations we really need to make sure the information is socialized and spread as much as possible and make sure we can preempt any sort of FUD.

Author: It might be good to add this in general (connecting the comms teams). Can you suggest the best ongoing point of contact for that (slack, email, attending a meeting, etc)? I will add something in both here, and maybe a note to reach out and intro at the cycle startup as well.

Author: We can also add this later. I'd really like to get the bulk of this merged if possible, as the release is starting up. Would we be ok merging it as it is now?

Reviewer: The current version looks good to me. I would be comfortable shipping this now and amending it later as needed.

Reviewer 5: I think it would be OK to do a follow up with those contacts. @mickeyboxell can you take an action (either for you or one of your shadows) to make that update?

Reviewer: Yep can do.

Regardless of reason, the timing for publication is ahead of the release, in order to inform the community and allow for preparation time. Start the discussion mid-cycle and well ahead of Code Freeze, and target publication for Code Freeze week.

### Press and release webinar

This is a light but very important component of the Communications Coordinator role. Two sets of activities need to be scheduled with the CNCF, namely press release and interview scheduling around the release day, and the release webinar after the release.

To schedule the press briefings, you will be a liaison between the Release lead and the CNCF contacts. Send an email about a month ahead of the release and coordinate between the parties to get release day press events scheduled.

> To schedule the release webinar with the CNCF, use the Calendly link (below) to schedule a "live webinar". This will typically occur 3-4 weeks after the release, and is primarily presented by the Release lead and Enhancements lead. Often the Comms lead will also join the webinar. The format is open, but primarily the team walks through the enhancements in the release and gives a sneak peek of what's coming in the next release.

Reviewer 6: In the past, I've just reached out to CNCF PR directly about this. Live webinars are pretty competitive to sign-up for, especially depending on the timing since they open up quarterly to projects and CNCF member companies. I'd also recommend reaching out maybe ~2 months beforehand (instead of 3-4 weeks) for the same timing issues. Thoughts?

Author: I'm cool with that. I've added emails for both pr and webinars. I think the timing is in the right place (4 weeks ahead of release to schedule a webinar 2-4 weeks after). Things look ok?

        <td></td>
        <td>
        <ul>
        <li>Work with Enhancements and Release Note leads to determine which deliverables are most noteworthy post-Enhancements Freeze
        <li>In the coming weeks, follow the progress of these enhancements, as they will roll into the Major Themes and be called out and described in the release blog

>         <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog (e.g. the <a href=https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/">1.22 Deprecations and Removals blog</a>

Reviewer 3: ```suggestion
        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog (similar to the <a href="https://blog.kubernetes.io/2021/07/14/upcoming-changes-in-kubernetes-1-22/">1.22 Deprecations and Removals</a> article.
```

        <li>Update `#sig-docs-blog` with final tracked feature blogs and establish a regular cadence status check-in with the team there
        </ul>
        </td>
    </tr>
    <tr>
        <td>12</td>
        <td></td>
        <td>
        <ul>
        <li>Begin the release blog draft, if not yet started
        <li>Optional 'Deprecations and Removals' blog ready for review
        <li>Schedule the release Live Webinar with CNCF via <a href="https://calendly.com/cncfonlineprograms">Calendly</a>, typically scheduled 3-4 weeks after the release
        <li>Schedule press and analyst pre-briefings and interviews for the release lead with CNCF
        <li>Schedule release blog and press embargo with CNCF
        </ul>
        </td>
    </tr>
    <tr>
        <td>13</td>
        <td><b>Code Freeze</b></td>
        <td>
        <ul>
        <li>Finalize and publish the 'Deprecations and Removals' blog once code freeze is in place.
        <li>Update release blog draft, post-Code Freeze
        <li>Check status on all feature blog PRs. Keep `#sig-docs-blog` up-to-date for editorial review, and establish tech reviewers are available 

>         <li>Check with Release Notes lead on content for Major Themes and Known Issues sections of the release blog
>         <li>Ensure that short one-to-two paragraph summaries of each Major Theme are available for the release blog from Release Notes lead or SIG Chairs

Reviewer 2: ```suggestion
```
Let's move this a week after Code Freeze. When Code Freeze starts, Release Notes will solicit Major Themes

Author: 👍
